### PR TITLE
Added 'row' block to table_for

### PR DESCRIPTION
**Pull request by jblackman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,9 @@ You'd have in your `app/views/foo/index.html.haml`:
 
 ```haml
 = table_for @foos [header_visible: (true|false), body_visible: (true|false), footer_visible: (true|false), downloadable: (:xlsx|:csv)] do
+  - row do |attributes,foo|
```
Comment (lleirborras) [inline README.md]: Can we consider also the case that we want to add an static class to the whole row by doing it with an `{}` parameter? like
```ruby
- row class: 'foo' do |attributes, foo|
  attributes['data-foo-id'] = foo.id
```

wdyt?

Comment (vjt) [inline README.md]: mh, we are already adding attributes through the block - based on the object that is being rendered at the moment - so passing other attributes through parameters to `row` looks duplicate to me.

```ruby
- row do |attributes, foo|
  attributes['class'] = 'frobber'

Comment (vjt) [inline README.md]: yes it looks verbose, on that I agree.

**Comment by vjt** [comment]
Merging for now, further API improvements are welcomed :-)

**Pull request merged**